packages/fuel-indexer/src/executor.rs

```diff
@@ -488,7 +495,15 @@ where
         pool: IndexerConnectionPool,
         handle_events_fn: fn(Vec<BlockData>, Arc<Mutex<Database>>) -> F,
     ) -> IndexerResult<Self> {
-        let db = Database::new(pool, manifest).await?;
+        let mut db = Database::new(pool.clone(), manifest).await;
+        let mut conn = pool.acquire().await?;
+        let version = fuel_indexer_database::queries::type_id_latest(
```

Comment:
Unrelated to this PR, but as you previously mentioned `Manifest::graphql_schema` should produce a `GraphQLSchema` from which we can get the `version` by just calling `GraphQLSchema::version`.So this could eventually be updated to:

```rust
let version = manifest.graphql_schema().version();
```